### PR TITLE
fix bug on character bonus tiles

### PR DESCRIPTION
**Pull request by cboki**
Hey @kamilmielnik 

thanks a lot for your help and your reactivity with the issue #112. I tried to fix the bug with your indications and it worked.

Capture d’écran 2021-04-07 155326
Capture d’écran 2021-04-07 155403

### CODE REVIEW

File: packages/types/src/CharacterBonus.ts [hunk 1]
```diff
@@ -13,6 +13,10 @@ class CharacterBonus extends Bonus {
   }
 
   public matchesCellTileScore(config: Config, cell: Cell): boolean {
+    if(typeof this.score === 'undefined') {
```
Comment (kamilmielnik) [inline packages/types/src/CharacterBonus.ts:16]: There is a code formatting error here:

```suggestion
    if (typeof this.score === 'undefined') {
```

**Review by kamilmielnik — CHANGES_REQUESTED** [review]
Thank you for the PR @cboki, good job!

However, there is a code formatting error - please fix it and we're good to go!

I have [ensured](https://github.com/kamilmielnik/scrabble-solver/commit/73523e62441a1696bc112906520bfdc46effd4e3) that GitHub Actions will run for pull requests coming from forks (like this one) so now you can see the problem at the bottom of pull request:

image

**Comment by cboki** [comment]
@kamilmielnik thanks for your answer. Here is the missing space. 👍

**Review by kamilmielnik — APPROVED** [review]
Thank you !

**Pull request merged**